index.html

```diff
@@ -10,55 +10,50 @@
 </head>
 
 <body>
-    <!-- home page header -->
+    <header>
+        
+        <!-- Navigation  bar containr-->
+        <nav class="navbar">
+            <a class="nav-logo" href="">My Logo</a>
+            <!-- static menu button -->
+            <ul class="nav-menu">
+                <li class="nav-item"><a class="nav-link" href="#portfolio-section" onclick="closeItem()">Portfolio</a></li>
+                <li class="nav-item"><a class="nav-link" href="#about-section" onclick="closeItem()">About</a></li>
+                <li class="nav-item"><a class="nav-link" href="#contact-section" onclick="closeItem()">Contact</a></li>
+            </ul> 
+            <div class="hamburger">
+               <span class="bar"></span>
+               <span class="bar"></span>
+               <span class="bar"></span>
+            </div>                    
+        </nav>  
+    </header>
```

Comment:
- [ ] kindly fix the style of the mobile menu to match Figma design 
- font sizes are incorrect 
- missing the blue background
- wrong margins and padding

Your Menu          |  Figma Design
:-------------------------:|:-------------------------:
![MMENUU](https://user-images.githubusercontent.com/89282221/189908122-b542d7cc-b1fe-4587-ae45-f2d41bd42db2.gif) |  ![image](https://user-images.githubusercontent.com/89282221/189908245-7e0d415f-d050-41c5-bb50-664ed9358e46.png)